app/models/contest_group.rb

```diff
@@ -0,0 +1,5 @@
+class ContestGroup < ActiveRecord::Base
+  has_many :contests, dependent: :destroy
```

Comment:
Please, don't use `dependent: :destroy` here. If someone wants to delete a content group with contests in the group there should be an error. We don't want someone to accidentally delete a lot of contests with a single click.